src/DateInput.jsx

```diff
@@ -216,6 +216,10 @@ export default class DateInput extends PureComponent {
     return getFormatter(options, locale);
   }
 
+  formatNumber(num, locale) {
```

Comment:
I'd prefer to have it the same way as `formatDate`, so a getter function. This should work:

```js
get formatNumber() {
  const { locale } = this.props;
  return num => num.toLocaleString(locale, { useGrouping: false });
}
```

and use:

```this.formatNumber(num)```

Reply:
thanks for the suggestion!